packages/scripts/preset/README.md

```diff
@@ -0,0 +1,120 @@
+# Preset Talend
+
+This is a preset for @talend/scripts. It holds most of Talend projects tools configuration.
+
+This preset allows some customisation through specific entry points. The configuration is done via `Talend/scripts` configuration file.
+
+## Configuration overview
+
+```json
+{
+  "preset": "talend",
+  "html": {
+    "title": "Talend Data Preparation",
+    "otherOptions": "Option value passed to html-webpack-plugin"
+  },
+  "sass": {
+    "data": {
+      "$brand-primary": "#4F93A7",
+      "$brand-primary-t7": "#00A1B3",
+      "$brand-secondary-t7": "#168AA6"
+    },
+    "theme": "tdp"
+  },
+  "webpack": {
+    "config": {
+      "development": "./webpack.config.dev.js",
+      "production": "./webpack.config.prod.js"
+    },
+    "api-url": "http://localhost:3030"
```

Comment:
Camel case or slugify, we have to choose once :)

Reply:
👍 the camel case was just an example, but the specific managed options are in kebab-case.
I changed the custom options example